packages/shared/src/domain/contact/__tests__/email.test.ts

```diff
@@ -0,0 +1,45 @@
+import { isEmailValid, normalizeEmail, normalizeEmailStrict } from "../email";
+
+describe("Email Utility Functions", () => {
+  describe("isEmailValid", () => {
+    it("should return false for empty string", () => {
+      expect(isEmailValid("")).toBe(false);
+    });
+
+    it("should return false for invalid email format", () => {
+      expect(isEmailValid("invalid-email")).toBe(false);
+    });
+
+    it("should return true for valid email", () => {
+      expect(isEmailValid("test@example.com")).toBe(true);
+    });
+
+    it('should return false for email with "mailto:" prefix', () => {
+      expect(isEmailValid("mailto:test@example.com")).toBe(false);
```

Comment:
added tests so we understand that at present cases like this will not be considered valid emails